routes/tasks.go

```diff
@@ -0,0 +1,111 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+
+	m "github.com/kirikami/go_exercise_api/database/models"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) SaveTaskHandler(c echo.Context) error {
+	task := m.Task{}
+
+	if err := c.Bind(&task); err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	if err := h.DB.Save(&task).Error; err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
```

Comment:
If your app can't save or do something after request validation, this is not `StatusBadRequest ` this is internal server error. Please update in all handlers accordingly